c2corg_ui/format/img.py

```diff
@@ -0,0 +1,89 @@
+'''
+c2corg img Extension for Python-Markdown
+==============================================
+
+Converts tags like
+[img=<id>(<options>)(/)?](<caption>)?([/img])?
+to advanced HTML img tags.
+'''
+
+from markdown.extensions import Extension
+from markdown.inlinepatterns import Pattern
+from markdown.util import etree
+
+
+class C2CImageExtension(Extension):
+
+    def __init__(self, *args, **kwargs):
+        self.config = {
+            "api_url": ['', 'Base URL of the API. Defaults to ""']
+        }
+
+        super(C2CImageExtension, self).__init__(*args, **kwargs)
+
+    def extendMarkdown(self, md, md_globals):  # noqa
+        self.md = md
+
+        IMG_RE = r'\[img=(\d+)([a-z_ ]*)(/\]|\](.*?)\[/img\])'  # noqa
```

Comment:
I have done as in https://github.com/waylan/Python-Markdown/blob/master/markdown/extensions/wikilinks.py#L44-L48

but in some other extensions it's done as you suggested: https://github.com/waylan/Python-Markdown/blob/master/markdown/extensions/nl2br.py#L24

I will make the change for the current extension and also for the other extension we have added (C2CWikilink).